src/rgw/rgw_notify.cc

```diff
@@ -813,87 +825,103 @@ int publish_reserve(const DoutPrefixProvider *dpp, EventType event_type,
   return 0;
 }
 
-int publish_commit(rgw::sal::Object* obj,
-        uint64_t size,
-        const ceph::real_time& mtime, 
-        const std::string& etag, 
-        const std::string& version,
-        EventType event_type,
-        reservation_t& res,
-        const DoutPrefixProvider *dpp) 
+extern int publish_commit(rgw::sal::Object* obj,
```

Comment:
what does "extern" mean here?

Reply:
merely that it's external--that's the default anyway, I was going to remove that